### ISSUE TITLE
Set timezone in Liberty container

### ISSUE THREAD

**Issue by faandg** [WASdev/ci.docker#359]
Greetings,

We noticed there is a tzdata package installed by default in the container, yet we did not find a way to properly change the timezone.
Is there a recommended way to do this (dockerfile run command / arg available / ...)?
In our case we are using OpenShift, but you might have a generic solution.

Thanks in advance.

**Comment by arthurdm** [comment]
hey @arturdzm - can you please take a look into this one?

**Comment by faandg** [comment]
Same question for locale.

**Comment by faandg** [comment]
Not sure if this is the way to go, but currently we're going with setting timezone and locale in the Dockerfile;

```
USER root 
RUN dnf -y remove glibc-minimal-langpack \
 && dnf -y install glibc-locale-source glibc-all-langpacks \
 && localedef --force -i nl_BE -f UTF-8 nl_BE.UTF-8 \
 && dnf clean all 
ENV LANG nl_BE.UTF-8 
ENV LC_ALL nl_BE.UTF-8
ENV TZ Europe/Brussels
```

**Comment by bbarman4u** [comment]
@arturdzm or any one is there an official answer on it for the ubuntu and UBI based images respectively ?

**Comment by arturdzm** [comment]
Yes, this is general solution, the language packs must be installed on top of original image and if there are multiple you could switch locales at runtime with ENV variables.
For ubuntu based images the language packs can be installed by `apt-get install language-pack-nl` for example.

**Comment by faandg** [comment]
@arthurdm @arturdzm would it be possible to add some information about this to the readme?
I think this is generally the place where people expect to find something about TZ or LANG settings (as some image creators choose to make them configurable).
